src/plugins/intel_gpu/src/plugin/plugin.cpp

```diff
@@ -233,6 +239,9 @@ ov::SoPtr<ov::IRemoteContext> Plugin::get_default_context(const AnyMap& params)
     if (params.find(ov::device::id.name()) != params.end())
         device_id = params.at(ov::device::id.name()).as<std::string>();
 
+    auto context = get_default_context(device_id);
+    context->initialize();
```

Comment:
what about including `initialize` into `get_default_context`? maybe we can add additional argument `bool initialize = true` for general case.

Reply:
Updated with default `bool initialize = true` value